sdks/python/apache_beam/options/pipeline_options.py

```diff
@@ -438,6 +438,12 @@ def _add_argparse_args(cls, parser):
         type=int,
         default=1,
         help='number of parallel running workers.')
+    parser.add_argument(
+        '--direct_running_mode',
+        default=None,
```

Comment:
Would it make sense to add another choice for `EmbeddedWorkerHandler` and change the default to that, instead of defaulting to `None`?

Reply:
Yes, that makes more sense, I will make a commit.